src/components/Community/data.json

```diff
@@ -44,26 +44,26 @@
     }
   ],
   "userContent": [
+    {
+      "url": "https://medium.com/acing-ai/ml-ops-data-science-version-control-5935c49d1b76",
+      "title": "ML Ops: Data Science Version Control",
+      "author": "Vimarsh Karbhari",
+      "date": "2020-04-23",
+      "pictureUrl": "/img/community/ugc/medium_logo.png"
+    },
+    {
+      "url": "https://blog.codecentric.de/en/2020/01/remote-training-gitlab-ci-dvc/",
+      "title": "Manage your Data Science Project in R",
+      "author": "Marcel Ribeiro-Dantas",
+      "date": "2020-03-05",
+      "pictureUrl": "/uploads/avatars/marcel.jpg"
+    },
     {
       "url": "https://blog.codecentric.de/en/2020/01/remote-training-gitlab-ci-dvc/",
       "title": "Remote training with GitLab-CI and DVC",
       "author": "Marcel Mikl and Bert Besser",
       "date": "2020-01-28",
       "pictureUrl": "/img/community/ugc/codecentric.png"
-    },
-    {
-      "url": "https://martinfowler.com/articles/cd4ml.html",
-      "title": "Continuous Delivery for Machine Learning",
-      "author": "Danilo Sato, Arif Wider and Christoph Windheuser",
-      "date": "2019-09-20",
-      "pictureUrl": "/img/community/ugc/fowler_icon.ico"
```

Comment:
@andronovhopf  we can delete these images?

Reply:
yep

Reply:
done

Reply:
🙏